temoa/extensions/method_of_morris/morris_evaluate.py

```
@@ -49,19 +55,14 @@ def evaluate(param_info, mm_sample, data, i, config: TemoaConfig, log_queue, log
     log_entry = ['']
     for j in range(0, len(mm_sample)):
```

Ruff (0.14.10)

Unnecessary `start` argument in `range`

Remove `start` argument

(PIE808)